README.md

```diff
@@ -1,26 +1,26 @@
-# DEVELOPING FOR MAFIA IN JS
+# DEVELOPING FOR MAFIA IN TYPESCCRIPT (TS)
```

Comment:
i think you mean TYPESCRIPT
![image](https://user-images.githubusercontent.com/8014761/183272468-a8ec7c81-0616-4896-8175-1b44ed38a712.png)